Source/restrict.cpp

```diff
@@ -39,6 +47,10 @@ BOOL RestrictedTest()
 	return ret;
 }
 
+/**
+ * @brief Check that we have write access to the game install folder
+ * @return True if we have write access

```

Comment:
```suggestion
 * @return False if we have write access

```

Reply:
This is terribly illogic, glad we have this documented now :D